lib/fastlane/plugin/wpmreleasetoolkit/actions/common/buildkite_trigger_build_action.rb

```diff
@@ -23,7 +23,11 @@ def self.run(params)
           options
         )
 
-        response.state == 'scheduled' ? UI.message('Done!') : UI.crash!("Failed to start job\nError: [#{response}]")
+        if response.state == 'scheduled'
+          UI.message("Successfully scheduled new build. You can see it at '#{response.web_url}'")
```

Comment:
See https://buildkite.com/docs/apis/rest-api/builds#create-a-build for API response documentation